bin/setup.sh

```diff
@@ -1,12 +1,22 @@
 #!/bin/bash
 
 echo
-echo "What's the full URL of your headless front-end repo, ending in .git?"
+echo "In your headless front-end repo, what is the value of \"name\" in its root package.json?"
+echo "For example, the \"name\" of this is \"getting-started\": https://github.com/kienstra/headless-wordpress/blob/ee93bcfbb72c446b78e09c8687c13361e258b1fc/package.json#L2"
```

Comment:
It'd be better if we didn't have to ask for this, and could get it when they give us the repo URL